react-native/training/src/app/screens/Home/screens/Book/components/BookDetail/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { ScrollView, Image, View } from 'react-native';
+import PropTypes from 'prop-types';
+
+import CustomText from '../../../../../../components/CustomText';
+
+import styles from './styles';
+
+const defaultImage = '../../../../../../../assets/book-cover-default.jpg';
+
+function BookDetail({ navigation }) {
+  const data = navigation.getParam('data');
+  const source = data.image_url ? { uri: data.image_url } : require(defaultImage);
```

Comment:
Thats a big problem in the performance. U are importing a file in each render. U need to import it once, and just use here

Reply:
Also, u can define this `source` once, in the component creation, so transform this component to a class and define this source as a prop of this class